patches/ts-transformer-inline-file+0.1.1.patch

```diff
@@ -0,0 +1,47 @@
+diff --git a/node_modules/ts-transformer-inline-file/transformer.js b/node_modules/ts-transformer-inline-file/transformer.js
```

Comment:
Ok, so webpacking without building first caused problems because `ts-loader` can't handle project references correctly and I had to disable the typeChecker for the final bundle. But then our transforms stopped working...because they rely on types.

By the time I figure that out, well... I had it figured out. So here is that file.

I'll probably just switch to requiring `tsc` to be run before `webpack` in the near future (which will allow us to use the super fast `esbuild`, since we should  just be compiling JS at that point). But I am committing this for now since I need to move on to other things.